examples/hcnn_compressed_example.py

```diff
@@ -0,0 +1,76 @@
+import sys, os
+
+sys.path.append(os.path.abspath(".."))
+sys.path.append(os.path.abspath("."))
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+from prosper_nn.models.hcnn_compressed import HCNN_compressed
+import prosper_nn.utils.generate_time_series_data as gtsd
+import prosper_nn.utils.create_input_ecnn_hcnn as ci
+
+# %% Define network parameters
+
+n_data = 10
+batchsize = 1
+n_batches = 2
+n_state_neurons = 10
+n_features_task = 2
+n_features_support = 5
+n_features_cSupport = 3
+past_horizon = 10
+forecast_horizon = 5
+
+
+# %% Create data and targets
+target_task = torch.zeros((past_horizon, batchsize, n_features_task))
+target_cSupport = torch.zeros((past_horizon, batchsize, n_features_cSupport))
```

Comment:
In the code, cSupport never appears. Maybe choose something more consistent?

Reply:
Also, does this mean I have to compress my features beforehand to get the target? But you pass the uncompressed support to the model to then compress it internally? I don't really get it.

Reply:
cSupport is the old naming of sup_comp -> Changed.

The target is just zeros (equal to task), but instead of using the full vector we use the compressed vector. Therefore our target also has to be in the compressed dimension.